client/state/i18n/locale-suggestions/reducer.js

```diff
@@ -0,0 +1,17 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { I18N_LOCALE_SUGGESTIONS_ADD } from 'state/action-types';
+
+export const items = ( state = null, action ) => {
```

Comment:
Non-blocking too but I think `localeSuggestions` makes more sense here, since it maps to `state.i18n.localeSuggestions`